app/src/components/users/AddUserForm.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <form @submit.prevent="addUser">
+    <TextInput
+      label="Name"
+      id="name"
+      :value="newUser.name"
+      @update:value="updateUser('name', $event)"
+      :isRequired="true"
+      pattern="[A-Za-z '\-]+"
+      title="Only letters, spaces, hyphens, apostrophes, and backticks are allowed"
+    />
+    <TextInput
+      label="Surname"
+      id="surname"
+      :value="newUser.surname"
+      @update:value="updateUser('surname', $event)"
+      :isRequired="true"
+      pattern="[A-Za-z '\-]+"
+      title="Only letters, spaces, hyphens, apostrophes, and backticks are allowed"
+    />
+
+    <DateInputField
+      label="Date of Birth"
+      id="dateOfBirth"
+      :value="newUser.dateOfBirth"
+      @update:value="updateUser('dateOfBirth', $event)"
+      :isRequired="true"
+      :minDate="minDate"
+      :maxDate="maxDate"
+    />
+
+    <TextInput
+      label="Position"
+      id="position"
+      :value="newUser.position"
+      @update:value="updateUser('position', $event)"
+      :isRequired="true"
+      pattern="[A-Za-z '\-]+"
+      title="Only letters, spaces, hyphens, apostrophes, and backticks are allowed"
+    />
+
+    <TextInput
+      label="Country"
+      id="country"
+      :value="newUser.country"
+      @update:value="updateUser('country', $event)"
+      :isRequired="true"
+      pattern="[A-Za-z '\-]+"
+      title="Only letters, spaces, hyphens, apostrophes, and backticks are allowed"
+    />
+
+    <div class ="modal-buttons">
+      <CustomButton styleType="secondary" type="button" title="Close" @click="closeClick"/>
+      <CustomButton type="submit" title="Add"/>
+    </div>
+  </form>
+</template>
+
+<script>
+import axios from 'axios';
+import CustomButton from "../common/Button.vue"
```

Comment:
```suggestion
import Button from "../common/Button.vue"
```
if you changed the name of the component, try not to leave the old name anywhere